product_docs/docs/pgd/5/node_management/maintainance_with_proxies.mdx

```diff
@@ -88,7 +88,7 @@ select read_nodes from bdr.node_group_routing_summary where write_lead = (select
 !!!
 
 ## Using `bdr.run_on_nodes()`
-PGD does have the ability to run specific commands on specific nodes using the `bdr.run_on_nodes()` function. This takes two parameters, an array of node names and the command you would like to run on those nodes. For example:
+PGD can run specific commands on specific nodes using the `bdr.run_on_nodes()` function. This function takes two parameters: an array of node names and the command you want to run on those nodes. For example:
```

Comment:
This is introducing people to the fact that there is this capability. It's not widely known.